src/app/views/NoteCreate.svelte

```diff
@@ -139,11 +140,28 @@
           ],
         }),
     })
+    const thunk = publish({
+      event: template,
+      relays: ctx.app.router.PublishEvent(template).getUrls(),
```

Comment:
This template doesn't have a pubkey yet, so random relays are selected. You'll need to sign the event first.

Reply:
Also, when using nip 46, there's a pretty long delay before a note gets published. This isn't really part of this PR, but go ahead and add loading to NoteCreate to avoid double publishes.